apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -39,7 +45,13 @@
                                 <<"channels.update_ack">>        -> update_ack;
                                 <<"channels.shutdown_sign">>     -> shutdown;
                                 <<"channels.shutdown_sign_ack">> -> shutdown_ack;
-                                <<"channels.leave">>             -> leave
+                                <<"channels.leave">>             -> leave;
```

Comment:
I see this `<<"channels.leave">>` is the only element in `METHOD_TAG` but not in `METHOD_SIGNED`. Is this intended?

Reply:
The `leave` action doesn't require signing.